<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 현재날짜출력</string>
    </property>
   </widget>
   <widget class="QLabel" name="clockIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>clock</string>
    </property>
   </widget>
   <widget class="QLabel" name="temIcon">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>tem</string>
    </property>
   </widget>
   <widget class="QLabel" name="humIcon">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>hum</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 26pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p *C</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 26pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p %</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weather_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>현재날씨상태</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>이야기 해요 ^_^</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
